src/actix/actix_telemetry.rs

```diff
@@ -1,19 +1,20 @@
+use std::future::{ready, Ready};
 use std::sync::Arc;
 
 use actix_web::dev::{Service, ServiceRequest, ServiceResponse, Transform};
 use actix_web::Error;
 use futures_util::future::LocalBoxFuture;
-use tokio::sync::Mutex;
+use parking_lot::Mutex;
```

Comment:
why is it necessary to use `parking_lot` here?

Reply:
Regarding this description of tokio mutex
https://docs.rs/tokio/latest/tokio/sync/struct.Mutex.html#which-kind-of-mutex-should-you-use
I'm using parking lot mutex where possible (but not everywhere). In this actix middleware we have 2 mutex-covered structures: for the first mutex is unique per working thread. for the second it is used only on actix start.